### ISSUE TITLE
Test for contrast issues

### ISSUE THREAD

**Issue by kimisgold** [omeka/theme-bigpicture#31]
Using accessibility audit tools, please check to see if contrast issues have been resolved on the following views:

* Home
* Resource browse views
* Resource show views
* Exhibit summary page

Test on desktop browsers with latest updates on `master` for both Big Picture and ExhibitBuilder.

**Comment by katknow** [comment]
I only found a few issues, which are:

- Home page - fine except for slider, but it's only getting picked up in Deque axe Tools and totally dependent on whatever image is the background for that particular slide
- Resource browse - "Title" and "Creator" and upper right & bottom right are being pinged as not having enough contrast, everything else seems to be fine. This is consistent with both Deque axe Tools and the other checkers.
- Resource show - Deque axe Tools pings everything on the page, but when I use the others the only thing getting pinged is the button for full screen in the light gallery viewer
- Exhibit summary page - this is coming up clear on the in-browser accessibility check, but Deque axe Tools is pinging the white text on the black background overlaid over the image banners

**Comment by kimisgold** [comment]
A bunch of good catches here. I'm going to give the home page and exhibit summary pages passes. The exhibit summary page clearly has white and black text on the title, so I think that's an issue with that particular tool's approach to calculating contrast. With the home page, I calculated the lightest gray value the white text could appear on (#fff on  #5a5a59), and the pairing passes the WCAG ratio.

The flags on the resource views were legitimate, and should now be fixed.

**Comment by katknow** [comment]
Interestingly, on the resource page the in browser accessibility tool is now flagging the buttons in the lightgallery viewer while Deque axe Tools is not. Other than that, the only thing still being highlighted as a problem in Deque axe Tools are the links under the Output Formats heading.

**Comment by kimisgold** [comment]
I've now made the buttons fully white on black. I'm still getting errors from the accessibility checker, but I suspect it's some error, same goes for the Output Formats. I think these are pretty visibly fixed.
